taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import Model, UniqueConstraint
+
+
+class Manufacturer(models.Model):
```

Comment:
if you imported the Model, why do you write models.Model? (3-4 str code)

Reply:
Yes you are right, it was written out of habit. Thank you. I corrected it.